Creating-Twitch-Account.md

```diff
@@ -26,6 +26,8 @@ On this page you can change your profile picture, email address and write a bio.
 
 ## Following FreeCodeCamp
 
+> We are currently moving away from using Twitch, we will update this page once details have been finalised.
```

Comment:
I know that *finalised* and _finalized_ both are correct, but *finalized* is the more common one, at least as far as English in US goes. It's the same _colour_ vs. *color* thing. Maybe you should use ~~finalised~~ **finalized**.